assets/prometheus-k8s/cluster-role.yaml

```diff
@@ -18,18 +18,6 @@ rules:
   - /metrics
   verbs:
   - get
-- apiGroups:
```

Comment:
just curious, why they are no longer useful?

Reply:
Because this PR also changes kube-rbac-proxy to use static authorizer. So It doesn't need to do SAR for each request to api-server to validate the Bearer token.

CC: @prashbnair

Reply:
Looks like we need these permissions after role because oauth-proxy also works with SARs.